### PR TITLE
Single kernel, package clean-up

### PR DESCRIPTION
**Pull request by cganley1**

### CODE REVIEW

**Comment by cganley1** [comment]
Included all frozen requirments, additional tests, some edits to env vars.

File: deployments/roman/environments/roman-cal/tests/imviz_simple_aper_phot.ipynb.disabled
Comment (msanchezst) [inline deployments/roman/environments/roman-cal/tests/imviz_simple_aper_phot.ipynb.disabled:1]: `.disabled` as extension? Is this what you wanted?

**Review by msanchezst — DISMISSED** [review]
Looks good to me. Thanks

File: deployments/roman/environments/roman-cal/tests/imviz_simple_aper_phot.ipynb.disabled
Comment (cganley1) [reply deployments/roman/environments/roman-cal/tests/imviz_simple_aper_phot.ipynb.disabled:1]: Yes. Headless test. You need a GUI to test jdaviz, so it's included for manual testing if needed.

Comment (msanchezst) [reply deployments/roman/environments/roman-cal/tests/imviz_simple_aper_phot.ipynb.disabled:1]: makes sense, thanks!

**Review by msanchezst — COMMENTED** [review]
Frozen requirements not included in PR?

**Comment by cganley1** [comment]
Updated, with new requirements and versions from document, and tested.

File: deployments/roman/Dockerfile.custom [hunk 1]
```diff
@@ -12,7 +12,7 @@ ENV CRDS_SERVER_URL=https://roman-crds.stsci.edu
 # ENV CRDS_VERBOSITY=20
 
 # Set for roman-cal phase 1 pointing to requested data in EFS
-ENV PYSYN_CDBS=/home/shared/preloaded-fits/phase1/reference-atlases
+ENV PYSYN_CDBS=/home/shared/preloaded-fits/phase1/reference-atlases/grp/redcat/trds/
```
Comment (msanchezst) [inline deployments/roman/Dockerfile.custom:15]: Updated path?

File: setup-env
Comment (msanchezst) [inline setup-env:1]: We probably don't want to include this file since it has hardcoded values

File: deployments/roman/Dockerfile.custom (hunk 1 above)
Comment (cganley1) [reply deployments/roman/Dockerfile.custom:15]: Yes, new path

**Review by msanchezst — APPROVED** [review]

**Comment by cganley1** [comment]
single kernel, package clean-up

**Pull request merged**